addons/mail/static/src/new/messaging.js

```diff
@@ -395,7 +430,7 @@ export class Messaging {
                 .call("mail.channel", "channel_fetch_preview", [ids])
                 .then((previews) => {
                     for (const preview of previews) {
-                        preview.last_message.date = markRaw(
+                        preview.last_message.date = toRaw(
```

Comment:
This should stay `markRaw`, as it's stored in state.

Reply:
Sorry, I changed the wrong one!